src/CreativeWork.ts

```diff
@@ -30,4 +30,7 @@ export default class CreativeWork extends Thing {
 
   @property('schema:version')
   version: Number | Text = ''
+
+  @property('schema:keywords')
+  keywords: Text = ''
```

Comment:
It's tempting to make this an `Array` but many APIs seem to use a comma separated list and schema.org says "Keywords or tags used to describe this content. Multiple entries in a keywords list are typically delimited by commas."